test/integration/acceptance/acceptance_fixture.hpp

```diff
@@ -162,8 +163,18 @@ class AcceptanceFixture : public ::testing::Test {
   const shared_model::crypto::Keypair kAdminKeypair;
   const shared_model::crypto::Keypair kUserKeypair;
 
+  const std::function<void(const shared_model::proto::TransactionResponse &)>
```

Comment:
You can declare the variables with just once type to avoid duplication:
`const std::function<void(const TxResponseType &)> checkEnoughSignatures, checkStatelessInvalid, ...`